tests/browser_test.go

```diff
@@ -141,3 +142,23 @@ func TestBrowserUserAgent(t *testing.T) {
 	}
 	assert.Contains(t, ua, "Headless")
 }
+
+func TestBrowserCrashErr(t *testing.T) {
+	t.Parallel()
+
+	defer func() {
+		robj := recover()
+		require.IsType(t, &goja.Object{}, robj)
+		gobj, _ := robj.(*goja.Object)
+		err, ok := gobj.Export().(error)
+		require.Truef(t, ok, "recovered object wasn't an error, but %T", err)
+		assert.Equal(t, "launching browser: Invalid devtools server port", err.Error())
+	}()
+
+	lopts := defaultLaunchOpts()
+	lopts.Args = []any{"remote-debugging-port=99999"}
+
+	newTestBrowser(t, lopts, withLogCache())
```

Comment:
`withLogCache` is not needed; leftover from some experiments.

```suggestion
	newTestBrowser(t, lopts)
```

Reply:
Resolved by f1a6a6d.